packages/serverpod/lib/src/server/session.dart

```diff
@@ -578,4 +578,9 @@ extension SessionInternalMethods on Session {
 
     return id;
   }
+
+  /// Returns true if the session is expected to be alive for an extended
+  /// period of time.
+  bool get isLongLived =>
+      this is StreamingSession || this is MethodStreamSession;
 }
```

Comment:
Suggestion: Use this as the check for the previous question I asked.

Here is an example of another attribute given to StreamingSessions, we should probably consolidate all of these so that another streaming session can be added by just modifying one place.